### PR TITLE
PMM-8432 remove i18n

### PR DESCRIPTION
**Pull request by gen1us2k**
PMM-8432

Build: SUBMODULES-0

- [ ] Links to other linked pull requests (optional).

### CODE REVIEW

**Review by percona-csalguero — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -113,7 +106,7 @@ env-up-start:
 		minikube config set kubernetes-version $(KUBERNETES_VERSION); \
 	fi
 	minikube config view
-	minikube start
+	minikube start --vm-driver=none
```
Comment (BupycHuk) [inline Makefile:109]: I'm not sure that it should be here

File: service/k8sclient/k8sclient_test.go [hunk 2]
```diff
@@ -264,6 +264,7 @@ func TestK8sClient(t *testing.T) {
 
 	var pmm *PMM
 	t.Run("PXC", func(t *testing.T) {
+		t.Skip("Skipping PXC. Times out on minikube")
```
Comment (BupycHuk) [inline service/k8sclient/k8sclient_test.go:267]: let's revert all these skips for now.

Comment (gen1us2k) [reply service/k8sclient/k8sclient_test.go]: Reverted

**Review by BupycHuk — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -84,15 +89,15 @@ jobs:
...
-          sudo -E minikube config set memory 4096
+      # - name: Configure minikube
+      #   run: |
+      #     sudo apt install -y conntrack
+      #     sudo curl -Lo /tmp/minikube https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64
+      #     sudo chmod +x /tmp/minikube
+      #     sudo mv /tmp/minikube $(which minikube)
+      #     sudo -E minikube config set driver none
+      #     sudo -E minikube config set cpus 2
+      #     sudo -E minikube config set memory 4096
```
Comment (BupycHuk) [inline .github/workflows/ci.yml:100]: I don't see replacement for minikube installation

**Pull request merged**